src/com/mambu/apisdk/MambuAPIServiceFactory.java

```diff
@@ -79,7 +79,31 @@ public static MambuAPIServiceFactory getFactory(String domain, String username,
 	public static MambuAPIServiceFactory getFactory(Protocol protocol, String domain, String username,
 			String password) {
 
-		Injector injector = Guice.createInjector(new MambuAPIModule(protocol, domain, username, password));
+		Injector injector = Guice.createInjector(new MambuAPIModule(protocol, domain, username, password, "Mambu SDKv1.0"));
```

Comment:
It looks like this value is hardcoded across the application (MambuAPIFactory, MambuAPIServiceFactory), so it can be extracted as a constant.